src/platform/packages/shared/kbn-es/src/cli_commands/serverless.ts

```diff
@@ -120,6 +123,19 @@ export const serverless: Command = {
       );
     }
 
+    if (options.productTier) {
```

Comment:
`productTier` is optional with this change to minimise the changes needed in all the files that use `src/platform/packages/shared/kbn-es/src/serverless_resources/project_roles/security/roles.yml` for security serverless tests.